### ISSUE TITLE
apply('content'), apply('tag') and others must return default values from `this.ctx`

### ISSUE THREAD

**Issue by nik-kor** [bem/bem-xjst#236]
```js
  it('should be ok', function() {
    test(function() {
      block('b1').def()(function() {
        return applyCtx([apply('content'), this.ctx.content, apply('tag'), this.ctx.tag])
      });
    }, { block: 'b1', tag: 'a', content: '1' },
      'aa11');
  });
```

Проверяли и на версии 6.2

http://bem.github.io/bem-xjst/?bemhtml=%0Ablock(%27b1%27).def()(function()%20%7B%0A%20%20%20%20return%20applyCtx(%5Bapply(%27content%27)%2C%20apply(%27tag%27)%2C%20%27xxx%27%2C%20this.ctx.tag%5D)%0A%7D)%3B&bemjson=%5B%7B%20block%3A%20%27b1%27%2C%20tag%3A%20%271%27%2C%20content%3A%20%27hahaha%27%7D%5D

**Comment by miripiruni** [comment]
Critical

**Comment by awinogradov** [comment]
@miripiruni we can close it, right?

**Comment by miripiruni** [comment]
Fixed in [v6.5.2](https://github.com/bem/bem-xjst/releases/tag/v6.5.2).
